wagtail/admin/templates/wagtailadmin/widgets/chooser.html

```diff
@@ -8,7 +6,7 @@
     when clicked.
 {% endcomment %}
 
-<div id="{{ attrs.id|unlocalize }}-chooser" class="chooser {% block chooser_class %}page-chooser{% endblock %} {% if not value %}blank{% endif %}">
```

Comment:
I don't really understand why `unlocalize` was here in the first place so I don't think it would be an issue to remove it but what's the reasoning behind it? Opportunistic cleanup?

This also applies to `wagtail/contrib/table_block/templates/table_block/widgets/table.html`.

Reply:
Yeah, this commit is a bit out of place due to me abandoning this strand of work at this point :-) This was going to be the first step in refactoring chooser.html to work with both the old-style and new-style template rendering, but I gave up on that idea and so this remains as a bit of standalone cleanup.

(As for why `unlocalize` was here in the first place: it was erroneously added in #3478 alongside various legitimate fixes to prevent numeric object IDs from being rendered with local number formatting.)